provider/src/main.ts

```diff
@@ -130,7 +131,8 @@ const init = async () => {
       }),
     }),
   )
-  map.addControl(new NavigationControl(), 'top-right')
+  map.addControl(new NavigationControl({}), 'top-right')
```

Comment:
Is this line's diff needed?

Reply:
I just tried to add some options of NavigationControl to see how it looks like. I forgot to delete it. 
Now I removed.